java/openmldb-jdbc/src/main/java/com/_4paradigm/openmldb/sdk/impl/SqlClusterExecutor.java

```diff
@@ -121,8 +126,7 @@ public boolean executeInsert(String db, String sql, SQLInsertRows rows) {
             logger.error("executeSQL fail: {}", status.getMsg());
         }
         status.delete();
-        status = null;
```

Comment:
@dl239 Why didi we use `status = null` after `status.delete()`?

Reply:
I have not tested whether it has impact on memory release  if it is not set to null.  In theory, it does not need to be set to null  because `status.delete()` can release the memory alloced in `C++`.